src/tests/integration.test.ts

```diff
@@ -392,14 +392,27 @@ describe.each(clients)('integration ($type)', ({ client }) => {
       expect(post).toBeNull()
       return
     }
-    // Should be unreacheable
+    // Should be unreachable
     const reached = true
     expect(reached).toBe(false)
   })
+
   test("Doesn't work with the Fluent API", async () => {
     const posts = await client.user.findUnique({ where: { email } }).posts()
     for (const post of posts!) {
       expect(post.content).toMatch(cloakedStringRegex)
     }
   })
+
+  test("query entries with non-empty name", async () => {
+    await client.user.create({
+       data: {
+         name: '',
+         email: 'test_mail@example.com'
+      }
+    });
```

Comment:
**suggestion**: We could also test that querying a `not: {existing email}` returns all the others records.